test/test_transformers.py

```diff
@@ -1444,6 +1444,7 @@ def test_mem_efficient_fail_bfloat16_sm50(self, device):
                     q, k, v, None, 0.0, False))
 
     @onlyCUDA
+    @unittest.skip("mem efficient flash attention might already be supported on sm90 and this test is out of date")
```

Comment:
I think we can go ahead and delete this test, as is it's not really worth keeping around IMO

Reply:
agreed, since both now support h100 this test can be removed

Reply:
Makes sense, removed